client.tsx

```diff
@@ -37,7 +43,7 @@ export function generateHeaders(options: IApolloClientGeneratorOptions) {
   return headers;
 }
 
-export interface IApolloClient<T> extends ApolloClient<T> {
+export interface IApolloClient<T> extends ApolloClient {
```

Comment:
Sweep: use BaseApolloClient instead of ApolloClient and pass the type parameter to BaseApolloClient